### DISCUSSION TITLE
Does skywalking have a performance test paper or some advice about the oap server?

### DISCUSSION THREAD

**Discussion by xiaozaiyuji** [apache/skywalking#11207] [Q&A]
We use skywalking with 12 cluster in product ,but I have some confusion about the performance of  skywalking oap server.As I try to improve the es write performance.I change the param as follows.
![image](https://github.com/apache/skywalking/assets/4230320/dcbf7170-6d5c-4745-b84d-67a86154b5e1)
But I find this is change very little with four instance in k8s cluster with 4c 12g 4 instance user 5 node elasticsearch 16c 64G 50000 iops  which support for 4000 agent.I monitor the es find the disk and memory is normal and the disk iops is only 25% and the kafka with 24 partions is normal too.
After that I increase the instance to 8 the es  iops increased to 65%. So I find the skywalking office website to find out how to solve this problem when the agent increase quickly and to increase the oap server instance or resource.But I did not find any document.So do we have some paper to help me? I need some advice and help.

**Comment by wu-sheng** [comment]
Officially we don't. You added some numbers, but we don't even know what it means. It is running on your own, and source codes are open. So, what costs the resource should be available for you?

We can't do magic.

**Reply by xiaozaiyuji** [reply]
I need some officecially explain.Although I have already solve this problem.
